include/seqan3/alphabet/detail/convert.hpp

```diff
@@ -38,8 +39,10 @@ constexpr std::array<out_t, alphabet_size<in_t>> convert_through_char_representa
 {
     [] () constexpr
     {
+        static_assert(alphabet_size<in_t> <= std::numeric_limits<size_t>::max());
```

Comment:
`
A type whose size cannot be represented by std::size_t is ill-formed (since C++14)
`
(https://en.cppreference.com/w/cpp/types/size_t)

So we do not need this